lib/mayaUsd/commands/PullPushCommands.cpp

```diff
@@ -404,6 +408,25 @@ MStatus DuplicateCommand::doIt(const MArgList& argList)
 
         auto& manager = PXR_NS::PrimUpdaterManager::getInstance();
         status = manager.duplicate(fSrcPath, fDstPath, userArgs) ? MS::kSuccess : MS::kFailure;
+
+        // If the duplicate src is Maya, the duplicate child of the destination
+        // will be USD, and vice-versa.  Construct an appropriate child path:
+        // - Maya duplicate to USD: we always duplicate directly under the
+        //   proxy shape, so add a single path component USD path segment.
+        // - USD duplicate to Maya: no path segment to add.
+        Ufe::Path childPath = (fSrcPath.runTimeId() == getMayaRunTimeId())
+            ?
+            // Maya duplicate to USD
```

Comment:
Really small nit-pick, if the comment was before the '?' it would layout better.